ingeniamotion/wizard_tests/base_test.py

```diff
@@ -52,6 +53,15 @@ def restore_backup_registers(self):
                 except IMRegisterWrongAccess as e:
                     self.logger.warning(e, axis=subnode)
 
+    @Stoppable.stoppable
+    def show_error_message(self):
+        error_code, axis, warning = self.mc.errors.get_last_buffer_error(
+            servo=self.servo, axis=self.axis)
+        _, _, _, error_msg = self.mc.errors.get_error_data(
```

Comment:
If you only want to get the last element of the tuple, you could do this `*_, error_msg = self.mc.errors.get_error_data()`